identity/webapp/src/app.ts

```diff
@@ -96,6 +96,11 @@ export async function createApp(): Promise<Koa> {
     ctx.respond = false;
   });
 
+  app.use(async (ctx, next) => {
```

Comment:
The identity app was behaving strangely following the upgrade, which was because everything was being served with a status code of 404 even though the response body was correct.

The content app was fine and [we were explicitly setting the status code there](https://github.com/wellcomecollection/wellcomecollection.org/blob/0cdd6d77f2dcc203f07e4bde929cff4e1e4e3834/content/webapp/app.ts#L94§), so I've done the same here.

Reply:
That's a bit of a strange one, do we know why it has to be done? (worth adding a comment?)

Reply:
I don't, but [it appears others have had a similar issue](https://github.com/vercel/next.js/issues/65691). I'll leave a comment